public/data/money8203.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Manish Tyagi",
+  "type": "personal",
+  "bio": "Techology Optimist | Open Source Enthusiast",
```

Comment:
```suggestion
  "bio": "Technology Optimist | Open Source Enthusiast",
```

Reply:
@kabir0x23  Thanks for clearly reviewing my code